src/main/scala/org/broadinstitute/hail/utils/richUtils/Implicits.scala

```diff
@@ -71,4 +72,23 @@ trait Implicits {
 
   implicit def toRichStringBuilder(sb: mutable.StringBuilder): RichStringBuilder = new RichStringBuilder(sb)
 
+  implicit def toTruncatable(s: String): Truncatable = new Truncatable {
+    def strings(delim: String, toTake: Int): (String, String) = {
```

Comment:
Does `delim` make sense here?  It would be slightly simpler to have the `delim` and `toTake` be determined by the `Truncatable`.  It certainly works for the case you have here and you can delete a bunch of variants of `fatal`, etc.